package.json

```diff
@@ -19,10 +19,11 @@
 	],
 	"dependencies": {
 		"@apollo/react-hooks": "^3.0.0",
+		"@viaa/avo2-components": "1.19.0",
 		"@viaa/avo2-types": "^2.2.1",
-		"@viaa/avo2-components": "1.17.0",
 		"apollo-boost": "^0.4.3",
 		"classnames": "^2.2.6",
+		"date-fns": "^2.0.0-beta.1",
```

Comment:
is dat een peer dependency voor react-datepicker?

Reply:
ja, dat was vanwege dit issue:
https://github.com/Hacker0x01/react-datepicker/issues/1815#issuecomment-513215416